tests/test_command_queue.h

```diff
@@ -276,50 +276,6 @@ TEST_CASE("[CommandQueue] Test Queue Basics") {
 			ProjectSettings::get_singleton()->property_get_revert(COMMAND_QUEUE_SETTING));
 }
 
```

Comment:
For the record the test below is no longer needed because the queue cant go full any longer.